com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.VirtualOffset.cs

```diff
@@ -17,6 +17,9 @@ private static void AddOccluders()
             for (int sceneIndex = 0; sceneIndex < UnityEngine.SceneManagement.SceneManager.sceneCount; ++sceneIndex)
             {
                 UnityEngine.SceneManagement.Scene scene = UnityEngine.SceneManagement.SceneManager.GetSceneAt(sceneIndex);
+                if (!scene.isLoaded)
```

Comment:
Oh, weird that a scene is in the scene manager but is not loaded, did not know that was possible :D

Reply:
Had that by just clicking on a scene in the hierarchy and click "Unload".